ui/app/src/components/PathNavigator/PathNavigator.tsx

```diff
@@ -247,7 +247,7 @@ export function PathNavigator(props: PathNavigatorProps) {
 
           if (withoutIndex(targetPath) === withoutIndex(path)) {
             // if path being deleted is the rootPath
-            dispatch(pathNavigatorRefresh({ id }));
+            dispatch(pathNavigatorBackgroundRefresh({ id }));
```

Comment:
Please retest how it behaves in this case. If the root path is deleted, essentially the whole content should change to the "Path doesn't exist error". In this case, skeleton may be in order.